### PR TITLE
Only show last 4 digits of cc number

### PR DESCRIPTION
**Pull request by aagavin**

### CODE REVIEW

File: hopkin/routes/customer.py [hunk 1]
```diff
@@ -51,7 +51,6 @@ def customer_profile_info() -> dict:
         'paymentInfo': {
             'name': request['paymentInfo']['name'],
             'cardType': request['paymentInfo']['cardType'],
```
Comment (aramrattan) [inline hopkin/routes/customer.py:57]: We are not sending the modified card num back with profile info? only when asked for payment info?

Comment (aagavin) [reply hopkin/routes/customer.py:57]: Thanks.

Good catch.

Comment (aagavin) [reply hopkin/routes/customer.py:57]: Fixed

**Review by aramrattan — APPROVED** [review]
Good job, looks good

**Review by LeontiBrechko — APPROVED** [review]
Yeah, it looks good! Sorry for the delay

**Pull request merged**